dashboard/modules/stats_collector/test_stats_collector.py

```diff
@@ -91,6 +91,83 @@ def getpid(self):
                 raise Exception(f"Timed out while testing, {ex_stack}")
 
 
+def test_memory_table(ray_start_with_dashboard):
```

Comment:
Can you move other memory table tests here? It'll be ideal if we can port all of them, but I'd like to at least port unit tests.

Reply:
The memory table tests are now in the `memory_utils` PR.